entries/val.xml

```diff
@@ -130,6 +130,9 @@ $( "input:text.items" ).val(function( index, value ) {
 });
       </code></pre>
       <p>This example appends the string " items" to the text inputs' values.</p>
+        <div class="warning">
+          <p><strong>Note: </strong> changing the value of an input element using this method won't fire the <code><a href="/change">change</a></code> event.</p>
```

Comment:
Do we need this in the `.val()` method as well? Lots of copy/pasted fragments make for a mess when it comes to updates. And yes we can turn these into reusable chunks but I'm questioning whether it's needed here since someone should be looking to event docs if they're listening for an event.